tests/test_views.py

```diff
@@ -1414,6 +1414,36 @@ def test_unsubscribe_mailchimp_not_called_if_user_not_subscribed(self, mock_get_
         assert_equal(mock_client.lists.subscribe.call_count, 0)
         handlers.celery_teardown_request()
 
+    def test_twitter_redirect_success(self):
+        self.user.social['twitter'] = 'someTwitterHandle'
+        self.user.save()
+        expected_redirect = 'https://twitter.com/someTwitterHandle'
+
+        res = self.app.get(web_url_for('redirect_to_twitter', uid=self.user._id))
+        assert_equals(res.status_code, 302)
+        assert_equal(res.location, expected_redirect)
+
+    def test_twitter_redirect_to_non_existent_user_returns_HTTPBad_request(self):
```

Comment:
This name is technically incorrect: a NOT FOUND *response* is returned. 

I'll make this change when I merge.